README.md

```diff
@@ -117,10 +115,16 @@ shell with:
 If you're running inside of an msys shell, however, you can run:
 
 ```sh
-$ ./configure --build=x86_64-pc-windows-msvc
-$ make && make install
+$ ./x.py build --build=x86_64-pc-windows-msvc
+$ ./x.py dist --build=x86_64-pc-windows-msvc --install
```

Comment:
It doesn't really matter whether Msys2 is a good choice or not. The fact is that with rustbuild the commands are identical no matter what shell you're in (aside from `python x.py` vs `./x.py`), so there's really no point in covering msys2 anymore. There was a point when this section mentioned you could use configure and make in msys2, but since it uses direct rustbuild now it provides no useful information.